BraveShared/Analytics/UrpService.swift

```diff
@@ -19,28 +18,23 @@ struct UrpService {
 
     let host: String
     private let apiKey: String
-    let sessionManager: Session
+    let sessionManager: URLSession
+    private let certificateEvaluator: PinningCertificateEvaluator
 
     init?(host: String, apiKey: String) {
         self.host = host
         self.apiKey = apiKey
 
-        guard let hostUrl = try? host.asURL(), let normalizedHost = hostUrl.normalizedHost else { return nil }
+        guard let hostUrl = URL(string: host), let normalizedHost = hostUrl.normalizedHost else { return nil }
 
         // Certificate pinning
-        let serverTrustPolicies: [String: PinnedCertificatesTrustEvaluator] = [
-            normalizedHost: PinnedCertificatesTrustEvaluator()
-        ]
-
-        sessionManager = Session(
-            serverTrustManager: ServerTrustManager(
-                evaluators: serverTrustPolicies
-            )
-        )
+        certificateEvaluator = PinningCertificateEvaluator(hosts: [normalizedHost])
```

Comment:
no host-validation?

Reply:
By default, `options` is set to `[.default, .validateHost]` in the constructor.